CODE_STYLEGUIDE.md

```diff
@@ -0,0 +1,174 @@
+# Code Style & Conventions
```

Comment:
I forgot all about these @arnoldsandoval-okta, thanks for bringing them back.

Reply:
I figured we may want to change/edit them, but I didn't see any harm in including them here

Reply:
Will remove.